lisa/operating_system.py

```diff
@@ -404,13 +404,24 @@ def uninstall_packages(
         package_names = self._get_package_list(packages)
         self._uninstall_packages(package_names, signed, timeout, extra_args)
 
-    def package_exists(self, package: Union[str, Tool, Type[Tool]]) -> bool:
+    def package_exists(
+        self,
+        package: Union[str, Tool, Type[Tool]],
+        minimum_version: Optional[VersionInfo] = None,
+    ) -> bool:
         """
         Query if a package/tool is installed on the node.
         Return Value - bool
         """
         package_name = self.__resolve_package_name(package)
-        return self._package_exists(package_name)
+        exists = self._package_exists(package_name)
+        if exists and minimum_version:
+            return (
```

Comment:
It needs to print debug message, if min version is not met. It saves time to troubleshooting the reason.